packages/tradershub/src/flows/RealAccountSIgnup/AccountOpeningSuccessModal/AccountOpeningSuccessModal.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import ReactModal from 'react-modal';
+import { useHistory } from 'react-router-dom';
+import Checkmark from '@/assets/svgs/checkmark.svg';
+import { ActionScreen, ButtonGroup } from '@/components';
+import { IconToCurrencyMapper } from '@/constants';
+import { CUSTOM_STYLES } from '@/helpers';
+import { ACTION_TYPES, useSignupWizardContext } from '@/providers/SignupWizardProvider';
+import { Button, Text, useDevice } from '@deriv-com/ui';
+
+const SelectedCurrencyIcon = () => {
+    const { state } = useSignupWizardContext();
+    return (
+        <div className='relative'>
+            <div className='[&>svg]:scale-[2.4] w-[96px] h-[96px] flex justify-center items-center'>
```

Comment:
@shahzaib-deriv just left a comment to change this class when the Quill-icons come to the place

Reply:
@mahdiyeh-deriv  added a comment for update the platform icon component